kmir/src/kmir/__main__.py

```diff
@@ -51,11 +51,11 @@ def exec_version(**kwargs: Any) -> None:
     print(f'KMIR Version: {VERSION}')
 
 
-def exec_init(llvm_dir: str, **kwargs: Any) -> KMIR:
-    print(
-        'WARN: "init" was seen in args, this calls an internal function. If a file is named "init", it must be renamed'
-    )
-    return KMIR(llvm_dir, llvm_dir)
+def exec_gen_glr_parser(llvm_dir: str, **kwargs: Any) -> None:
```

Comment:
This funtion is not need too?

Reply:
Maybe not, but then we are going to need to redesign the test runner (not opposed to that occurring but not sure it is a goal for this PR), which I am not sure how big a job that is. Right now we build everything manually in python as [fixtures](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html).

Reply:
Ignore this, I was wrong!